app/controllers/redirect/contact_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Redirect
+  class ContactController < RedirectController
```

Comment:
Would this be adding an unnecessary extra step to track redirects? I'm unsure about this.

Reply:
well, we do this in other places to ensure we have good tracking, so it's not completely unnecessary? but yes it is an extra step